rocAL/rocAL/source/meta_data/tf_meta_data_reader_detection.cpp

```diff
@@ -78,13 +78,13 @@ void TFMetaDataReaderDetection::lookup(const std::vector<std::string> &image_nam
         if(_map_content.end() == it)
         {
             _output->get_bb_cords_batch()[i] = {{0, 0, 0, 0}};
```

Comment:
why do we need to reset them to zeros?

Reply:
This part was changes in other metadata reader files to THROW error if data is not present, I guess this change was missed previously. Will make change to throw error in this PR